packages/store/src/services/store/operations/store-export.ts

```diff
@@ -36,7 +38,12 @@ export class StoreExportOperation implements StoreOperation {
     const apiShopId = await this.validateShop(sourceShopDomain)
 
     if (!flags['no-prompt']) {
-      if (!(await confirmExportPrompt(sourceShopDomain, toFile))) {
+      if (fileExistsSync(toFile)) {
+        if (!(await confirmOverwritePrompt(sourceShopDomain, toFile))) {
+          outputInfo('Exiting.')
+          process.exit(0)
+        }
+      } else if (!(await confirmExportPrompt(sourceShopDomain, toFile))) {
```

Comment:
I am slightly concerned that with file conflict we are getting less informative confirmation that is only asking to override the file and export prompt is not being shown. 

I would make it so export confirmation prompt is displayed first and then when user confirmed the intent to export from a store to a file we show the next prompt to confirm file overwrite.

nit, I would extract the confirmation logic into separate function since it becomes really bulky.

Reply:
Other issue I can see with it is that `no-prompt` mode will automatically overwrite the file. I would expect it to actually fail with the error stating that the file already exists. I think `no-prompt` should be still safe. E.g. when I use it in automation I would love it to not silently overwrite the file if I messed up with the logic. I don't have strong opinions on that so I would love to hear what other people think about.

Reply:
OK, I am good to have a single confirmation, I think it's more user friendly.

nit, I would love to see some kind of extracting this logic. Maybe it could be arguments passed to the same function `confirmExportPrompt`, or it could be another function. But not a blocker for this PR.

Also would love a change to avoid silent overwrite for `no-prompt` mode. But it could definitely be a follow up since this behavior is not changing in this PR and it's positive incremental change.